src/Tribe/PUE/Checker.php

```diff
@@ -568,6 +571,8 @@ public function validate_key( $key ) {
 					}
 				}
 
+				$pue_notices->clear_notices( $plugin_name );
```

Comment:
Just checking: are we positive we wish to clear notices at this point (during ajax validation, before a key has been committed/saved)?